manim/scene/scene.py

```diff
@@ -683,7 +683,7 @@ def get_time_progression(self, run_time, n_iterations=None, override_skip_animat
         time_progression = ProgressDisplay(
             times, total=n_iterations,
             leave=file_writer_config['leave_progress_bars'],
-            ascii=False if platform.system() != 'Windows' else True
+            ascii=True if platform.system() == 'Windows' else None
```

Comment:
I don't know anything about tdqm, but why do we have to use ascii on windows and another thing (unicode, iirc) on others OS ?

Reply:
some windows terminals lie about supporting unicode (see https://github.com/tqdm/tqdm/issues/454). Anyway the default should be `None` rather than `False` to auto-detect.